js/views/standalone_registration_view.js

```diff
@@ -47,7 +47,17 @@
       this.accountManager
         .registerSingleDevice(
           this.$('#mnemonic').val(),
-          this.$('#mnemonic-language').val()
+          this.$('#mnemonic-language').val(),
+          async (pubKey) => {
+            await storage.setProfileName(this.$('#display-name').val());
+
+            // Update the conversation if we have it
+            const conversation = ConversationController.get(pubKey);
```

Comment:
Is there a case where there is an existing conversation with ourself upon account creation?